### PR TITLE
cvmfs: use PKG_VERSION_xxx to test pkg_compare_version() return values

### PR DESCRIPTION
**Pull request by jouvin**
Requires https://github.com/quattor/template-library-core/pull/131 first... But would be good to merge it at the same time.

### CODE REVIEW

**Comment by jrha** [comment]
Have you tested that the results are identical?

File: features/cvmfs/rpms.pan [hunk 1]
```diff
@@ -5,10 +5,10 @@ unique template features/cvmfs/rpms;
 include 'quattor/functions/package';
 
 '/software/packages' = {
-  if ((pkg_compare_version(CVMFS_CLIENT_VERSION, '2.1.20') >= 0) && (OS_VERSION_PARAMS['major'] != 'sl5')) {
-    SELF[escape('cvmfs-config-default')] = dict();
-  } else {
+  if ((pkg_compare_version(CVMFS_CLIENT_VERSION, '2.1.20') == PKG_VERSION_LESS) || (OS_VERSION_PARAMS['major'] == 'sl5')) {
```
Comment (jrha) [inline features/cvmfs/rpms.pan]: This is not equivalent to the previous code, is it what was intended?

Comment (jouvin) [inline features/cvmfs/rpms.pan]: Yes it is (it should)! I inverted the test due to the problem described in https://github.com/quattor/template-library-core/issues/132. Before the test was `pkg_compare_version <= 0` which meant (if I am right) greater or equal. I swaped the if conditions and thus test for `less`.

**Comment by jouvin** [comment]
Yes it has been tested at GRIF and I saw no difference (as expected).

**Pull request merged**